sensors/simsensor_common/simsensor_reader.c

```diff
@@ -0,0 +1,270 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Simsensor reader reads and parses data form special scenario file
+ * than it pushes it to event_queue
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Piotr Nieciecki
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <stdbool.h>
+
+#include "simsensor_reader.h"
+
+#define READER_END_SCENARIO_INDICATOR NO_SENSOR
+#define READER_DATA_SEPARATOR         ','
+
+#define READER_TIMESTAMP_NOT_SET -1
```

Comment:
```suggestion
#define READER_TIMESTAMP_NOT_SET (time_t)-1
```

Reply:
I have decided to use `(time_t)LONG_MAX`, because time_t can be unsigned.

Reply:
Not too small a number?

Reply:
I think, that our purposes it's enough

Reply:
kemonats is right on this: LONG_MAX = INT_MAX = 2^31 ~= 2e8 µs, which is about half a hour. Better use LLONG_MAX.